analytics/scala/src/main/scala/com/koverse/example/spark/LogisticRegressionTraining.scala

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2016 Koverse, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.koverse.example.spark
+
+import org.apache.spark.ml.classification.LogisticRegression
+import org.apache.spark.ml.linalg.Vectors
+import org.apache.spark.ml.param.ParamMap
+import org.apache.spark.sql.{DataFrame}
+import com.koverse.sdk.Version
+import com.koverse.sdk.data.Parameter
+import com.koverse.sdk.transform.scala.{DataFrameTransform, DataFrameTransformContext}
+
+class LogisticRegressionTraining extends DataFrameTransform {
+
+    /*
+     * We’ll begin by defining a set of parameters that our Transform will use to request configuration information from a
+     * user of the Koverse UI. In this case we’ll ask the user to tell our Transform max iterations and regularization parameter.
+     */
+
+    final val MAX_ITERATIONS_PARAM = "maxIterations"
+    final val REGULARIZATION_PARAM = "regularizationParameter"
+    final val INPUT_DATASET = "inputDataSet"
+    final val TRAIN_LOGISTIC_REGRESSION_TRANSFORM = "logistic-regression-train"
+
+    def getParameters(): scala.collection.immutable.Seq[Parameter] = {
+
+//        val iterationParameter = new Parameter(, "Max Iterations", )
```

Comment:
If these commented out pieces are valuable please explain their value for users. Otherwise, you should delete them so they don't cause confusion.

Reply:
Roger that